Documentation/Ctrl/CtrlSearchFields.rst.txt

```diff
@@ -11,6 +11,8 @@ searchFields
     Comma-separated list of fields from the table that will be included when searching for records in the TYPO3 backend.
     No record from a table will ever be found if that table does not have "searchFields" defined.
 
+    Note that certain fields of type :ref:`input <columns-input>` are excluded by default.
```

Comment:
We should move the hint here including the mention of the hook.

Reply:
Please check the latest changes, if that's better

Reply:
Yep, sounds good now!